appveyor.yml

```diff
@@ -0,0 +1,27 @@
+build: false
+shallow_clone: false
+platform: 'x86'
+clone_folder: c:\projects\symfony
+environment:
+  global:
+    PHP: "C:/PHP"
+init:
+  - SET PATH=C:\php\;%PATH%
+install:
+  - cd c:\
+  - ps: Start-FileDownload 'http://ci.cakephp.org/php.zip'
+  - 7z x php.zip -oc:\php
+  - cd c:\php
+  - copy php.ini-production php.ini
+  - echo date.timezone="UTC" >> php.ini
+  - echo extension_dir=ext >> php.ini
+  - echo extension=php_openssl.dll >> php.ini
+  - echo extension=php_intl.dll >> php.ini
+  - echo extension=php_mbstring.dll >> php.ini
+  - echo extension=php_fileinfo.dll >> php.ini
+  - cd C:\projects\symfony
+  - php -r "readfile('https://getcomposer.org/installer');" | php
+  - php composer.phar install --prefer-dist --no-interaction
+test_script:
+  - cd C:\projects\symfony
+  - php -r "readfile('https://phar.phpunit.de/phpunit.phar');" | php
```

Comment:
what about using native ways to download files instead ? http://www.appveyor.com/docs/how-to/download-file